### PR TITLE
templace/ecr: AWS credentials not requiered

### PR DESCRIPTION
**Pull request by blackjid**
This enable support to use credentials resolved from the IAM Instance Profile if used. This was added to this project a few months ago. Also add reference to this in the readme

### CODE REVIEW

**Comment by blackjid** [comment]
hold this PR.... I just realized the author hasn't released the IAM profile feature to the docker hub

**Comment by blackjid** [comment]
waiting for https://github.com/objectpartners/rancher-ecr-credentials/issues/10

**Comment by cloudnautique** [comment]
@johnrengelman, can you let us know when this is good to go?

**Comment by johnrengelman** [comment]
Release as objectpartners/rancher-ecr-credentials:1.1.0

**Comment by johnrengelman** [comment]
This should probably add a new template version that uses the new 1.1.0 image instead of modifying the existing template.

**Comment by blackjid** [comment]
yes, when I opened the PR, I thought the functionality was in 1.0.1...

File: templates/ecr/README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Run this container with the following environment variables:
 * `AWS_ACCESS_KEY_ID`
 * `AWS_SECRET_ACCESS_KEY`
 
+If you're using an IAM instance profile, then you only need to specify the region,
+with the `AWS_REGION` environment variable, and the credentials will be
+automatically negotiated.
+
```
Comment (johnrengelman) [inline templates/ecr/README.md]: Can please update this file by just copying the contents from https://raw.githubusercontent.com/objectpartners/rancher-ecr-credentials/master/README.md

Comment (blackjid) [inline templates/ecr/README.md]: jej, I did that!! but the README.md changed a few minutes ago!.. :)

**Comment by blackjid** [comment]
I'll squash the commits when it's good to go

**Comment by johnrengelman** [comment]
@cloudnautique - LGMT

**Pull request merged**